src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -134,7 +152,8 @@ def Watch(self, request, context, send_response_callback=None):
                 self._on_close_callback(send_response_callback, service))
         return blocking_watcher
 
-    def set(self, service, status):
+    def set(self, service: str,
+            status: _health_pb2.HealthCheckResponse.ServingStatus) -> None:
```

Comment:
I think this should just be `int`. I also think this should have failed. Can you please make sure that type checking is working properly here? You may need to start generating type stubs for this package by modifying the corresponding `setup.py`.

Reply:
`_health_pb2.HealthCheckResponse.ServingStatus` was treated as `Any` thus Pytype didn't complain here.

Reply:
Ah, because of the lack of stubs. We need to make sure that the corresponding `_pb2.pyi` files get generated for each of the ancillary packages (and make sure that they're actually getting picked up too).